packages/launchpad/src/components/WizardLayout.vue

```diff
@@ -39,41 +38,26 @@ mutation WizardLayoutNavigate($direction: WizardNavigateDirection!) {
 }
 `
 
-export default defineComponent({
-  components: { ButtonBar },
-  props: {
-    next: {
-      type: String,
-      default: "Next Step",
-    },
-    back: {
-      type: String,
-      default: "Back",
-    },
-    alt: {
-      type: String,
-      default: undefined,
-    },
-    altFn: {
-      type: Function as PropType<(val: boolean) => void>,
-      default: undefined
-    }
-  },
-  setup(props) {
-    const navigate = useMutation(WizardLayoutNavigateDocument)
+const props = withDefaults(
+  defineProps<{
+    next: string
+    back: string
+    alt?: string
+    altFn?: (val: boolean) => void
+    nextFn?: (...args: unknown[]) => any
+}>(), {
+  next: 'Next Step',
+  back: 'Back Step',
```

Comment:
You accidentally added a word.
```suggestion
  back: 'Back',
```